src/lib/schematics/README.md

```diff
@@ -0,0 +1,12 @@
+# Angular Flex Layout Schematics
+A collection of Schematics for Angular Flex Layout.
+
+## Collection
+
+### Install
+Adds Angular Flex Layout and its dependencies and pre-configures the application. 
+
+- Adds Flex Layout and CDK to `package.json`
+- Ensure `FlexLayoutModule` is installed and included in root module
```

Comment:
Currently adding Flex-Layout is only supported for a specified application:

```console
ng add @angular/flex-layout --project=stellar-planner
```

We should add another option to add FlexLayout to a specified **module** using a `--module` option.

e.g.
```console
ng add @angular/flex-layout --module=libs/planner/feature-dashboard/src/lib/planner-dashboard.module.ts
```